mmv1/products/cloudrun/Service.yaml

```
@@ -850,6 +850,29 @@ properties:
                           name: 'sizeLimit'
                           description: |-
                             Limit on the storage usable by this EmptyDir volume. The size limit is also applicable for memory medium. The maximum usage on memory medium EmptyDir would be the minimum value between the SizeLimit specified here and the sum of memory limits of all containers in a pod. This field's values are of the 'Quantity' k8s type: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/. The default is nil which means that the limit is undefined. More info: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: csi
+                      description: |-
+                        A filesystem specified by the Container Storage Interface (CSI).
+                      min_version: beta
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'driver'
+                          required: true
+                          description: |-
+                            Unique name representing the type of file system to be created. Cloud Run supports the following values:
+                              * gcsfuse.run.googleapis.com: Mount a Google Cloud Storage bucket using GCSFuse
+                        - !ruby/object:Api::Type::String
```

```suggestion
                        - !ruby/object:Api::Type::Boolean
```

Currently failing tests with:

```
panic: template.0.spec.0.volumes.0.csi.0.read_only: '' expected type 'string', got unconvertible type 'bool', value: 'true'
```